### ISSUE TITLE
MDT / NBMolViz Remix

### ISSUE THREAD

**Issue by justinmc** [Autodesk/notebook-molecular-visualization#19]
This project should have a reasonable distinction between itself and [MDT](https://github.com/Autodesk/molecular-design-toolkit).  Right now it seems to be arbitrary.  Maybe the right way to divide them is that this would handle all ipywidgets stuff.

**Comment by avirshup** [comment]
Agreed! I think, based on our conversation today, we will have 4 packages like this (with randomly-chosen names):
- Molecular Design Toolkit: No frontend, no javascript, no notebook dependencies - just modeling routines
- MDTWidgets: MDT add-on package with all the notebook/frontend stuff. Depends on MDT and the following 2 javascript libraries
- ReactChemical2D: 2D javascript viewer
- ReactChemical3D: 3D javascript viewer
